pkg/remote/scanner.go

```diff
@@ -94,7 +95,7 @@ func (s *Scanner) scan() ([]*resource.Resource, error) {
 		return nil, err
 	}
 
-	if !s.options.Deep {
+	if !s.options.Deep && !s.options.OnlyManaged {
```

Comment:
See https://github.com/snyk/driftctl/pull/1359#issuecomment-1050886424

Reply:
I'd love to wait for @sjourdan answer about that.